UnitTest1.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace AutomationTut.by
+{ 
+    public class Tests
+    {
+       
+        private IWebDriver driver;
+      
+        private WebDriverWait wait;
+        public DateTime currentTime;
+
+        [SetUp]
+        public void Setup()
+        {
+           string baseURL = @"https://www.tut.by/";
```

Comment:
You can put it in one method called GotToWebsite() for example in BasePage and call it in test